app/helpers/courses_helper.rb

```diff
@@ -11,4 +11,9 @@ def pub_status_str(course)
     when "T" then "Trashed"
     end
   end
+
+  def course_topics(course)
+    topics = course.topics.map(&:title)
```

Comment:
There is a more concise and potentially performant way to get the array of topic titles:

```
course.topics.pluck(:title).join(", ")
```

The pluck, vs using map, will generate the sql query in ActiveRecord to return just the array of titles, not an array of ActiveRecord relation objects that you then have to loop over. While it won't make much a difference when looking over a single course's topics (as in this case, I doubt we will see more than a dozen or two), if dealing with a large number of database relation objects, it can make quite a difference in the performance.

The one line above can replace both lines of this method and offer better performance. It also leverages the implicit return of a method, so there is no need to then assign to a scoped variable.